[36m[2023-10-29 16:36:41 DEBUG] [cli:114][39m ###############################################################################
[36m[2023-10-29 16:36:41 DEBUG] [cli:115][39m Running command 'run'
[36m[2023-10-29 16:36:41 DEBUG] [cli:119][39m Using settings file '/home/muhammad_kik/.chaostoolkit/settings.yaml'
[36m[2023-10-29 16:36:44 DEBUG] [settings:30][39m The Chaos Toolkit settings file could not be found at '/home/muhammad_kik/.chaostoolkit/settings.yaml'.
[36m[2023-10-29 16:36:44 DEBUG] [__init__:445][39m No controls to apply on 'loader'
[36m[2023-10-29 16:36:44 DEBUG] [__init__:445][39m No controls to apply on 'loader'
[36m[2023-10-29 16:36:44 DEBUG] [caching:24][39m Building activity cache...
[36m[2023-10-29 16:36:44 DEBUG] [caching:35][39m Cached 3 activities
[32m[2023-10-29 16:36:44 INFO] [experiment:58][39m Validating the experiment's syntax
[36m[2023-10-29 16:36:44 DEBUG] [configuration:63][39m Loading configuration...
[36m[2023-10-29 16:36:44 DEBUG] [secret:78][39m Loading secrets...
[36m[2023-10-29 16:36:44 DEBUG] [secret:103][39m Done loading secrets
[32m[2023-10-29 16:36:44 INFO] [experiment:109][39m Experiment looks valid
[36m[2023-10-29 16:36:44 DEBUG] [caching:42][39m Clearing activities cache
[36m[2023-10-29 16:36:44 DEBUG] [cli:314][39m Runtime strategies: hypothesis - before-method-only / rollbacks - default
[36m[2023-10-29 16:36:44 DEBUG] [caching:24][39m Building activity cache...
[36m[2023-10-29 16:36:44 DEBUG] [caching:35][39m Cached 3 activities
[36m[2023-10-29 16:36:44 DEBUG] [configuration:63][39m Loading configuration...
[36m[2023-10-29 16:36:44 DEBUG] [secret:78][39m Loading secrets...
[36m[2023-10-29 16:36:44 DEBUG] [secret:103][39m Done loading secrets
[36m[2023-10-29 16:36:44 DEBUG] [configuration:160][39m Loading dynamic configuration...
[32m[2023-10-29 16:36:44 INFO] [run:369][39m Running experiment: Proxy Server is Offline
[36m[2023-10-29 16:36:45 DEBUG] [__init__:59][39m Initializing controls
[32m[2023-10-29 16:36:45 INFO] [run:396][39m Steady-state strategy: before-method-only
[32m[2023-10-29 16:36:45 INFO] [run:400][39m Rollbacks strategy: default
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'experiment'
[36m[2023-10-29 16:36:45 DEBUG] [run:563][39m Running steady-state hypothesis before the method
[32m[2023-10-29 16:36:45 INFO] [hypothesis:186][39m Steady state hypothesis: Normal Load Testing Check
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'hypothesis'
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'activity'
[32m[2023-10-29 16:36:45 INFO] [activity:215][39m Probe: Normal load testing log must exists
[36m[2023-10-29 16:36:45 DEBUG] [python:33][39m Activity 'Normal load testing log must exists' loaded from '/usr/lib/python3.8/genericpath.py'
[36m[2023-10-29 16:36:45 DEBUG] [activity:237][39m   => succeeded with 'True'
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'activity'
[36m[2023-10-29 16:36:45 DEBUG] [hypothesis:226][39m allowed tolerance is True
[32m[2023-10-29 16:36:45 INFO] [hypothesis:236][39m Steady state hypothesis is met!
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'hypothesis'
[32m[2023-10-29 16:36:45 INFO] [run:677][39m Playing your experiment's method now...
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'method'
[36m[2023-10-29 16:36:45 DEBUG] [__init__:445][39m No controls to apply on 'activity'
[32m[2023-10-29 16:36:45 INFO] [activity:215][39m Action: Retrieve all incidents repeatedly
[36m[2023-10-29 16:36:45 DEBUG] [python:33][39m Activity 'Retrieve all incidents repeatedly' loaded from '/home/muhammad_kik/.local/lib/python3.8/site-packages/chaosk6/actions.py'
[32m[2023-10-29 16:36:45 INFO] [actions:70][39m Stressing the endpoint "https://status.cloud.google.com/incidents.json" with 2 VUs for 5s.
[32m[2023-10-29 16:36:57 INFO] [actions:85][39m Stressing completed.
[32m[2023-10-29 16:36:57 INFO] [actions:87][39m Logged K6 output to log/running/k6.log.
[36m[2023-10-29 16:36:57 DEBUG] [activity:237][39m   => succeeded with 'True'
[36m[2023-10-29 16:36:57 DEBUG] [__init__:445][39m No controls to apply on 'activity'
[36m[2023-10-29 16:36:57 DEBUG] [__init__:445][39m No controls to apply on 'activity'
[32m[2023-10-29 16:36:57 INFO] [activity:215][39m Action: Run load testing when interface is off
[36m[2023-10-29 16:36:57 DEBUG] [python:33][39m Activity 'Run load testing when interface is off' loaded from '/home/muhammad_kik/.local/lib/python3.8/site-packages/chaosk6/actions.py'
[32m[2023-10-29 16:36:57 INFO] [actions:34][39m Running load-testing.js
[36m[2023-10-29 16:37:22 DEBUG] [activity:239][39m   => succeeded without any result value
[36m[2023-10-29 16:37:22 DEBUG] [__init__:445][39m No controls to apply on 'activity'
[36m[2023-10-29 16:37:22 DEBUG] [__init__:445][39m No controls to apply on 'method'
[32m[2023-10-29 16:37:22 INFO] [run:957][39m Let's rollback...
[36m[2023-10-29 16:37:22 DEBUG] [__init__:445][39m No controls to apply on 'rollback'
[32m[2023-10-29 16:37:22 INFO] [rollback:31][39m No declared rollbacks, let's move on.
[36m[2023-10-29 16:37:22 DEBUG] [__init__:445][39m No controls to apply on 'rollback'
[32m[2023-10-29 16:37:22 INFO] [run:515][39m Experiment ended with status: completed
[36m[2023-10-29 16:37:22 DEBUG] [__init__:445][39m No controls to apply on 'experiment'
[36m[2023-10-29 16:37:22 DEBUG] [__init__:98][39m Cleaning up controls
[36m[2023-10-29 16:37:22 DEBUG] [caching:42][39m Clearing activities cache
